docs/advanced-features/athena.ipynb

```diff
@@ -0,0 +1,688 @@
+   "id": "2179997c-399b-43a8-b280-aed4cf411e8a",
+   "metadata": {},
+   "source": [
+    "### A. Manifests table\n",
+    "The following Athena DDL will build a table of all the manifests in that bucket\n",
+    "(all package-level and object-level metadata).\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e9b4d2fa-6463-465b-8072-db45466ed596",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DDL[MANIFEST_TABLE] = f\"\"\"\n",
+    "CREATE EXTERNAL TABLE IF NOT EXISTS `{ATHENA_DB}.{MANIFEST_TABLE}`(\n",
+    "  `logical_key` string,\n",
+    "  `physical_keys` array<string>,\n",
+    "  `size` string,\n",
+    "  `hash` struct<type:string,value:string>,\n",
+    "  `meta` string,\n",
+    "  `user_meta` string,\n",
+    "  `message` string,\n",
+    "  `version` string)\n",
+    "ROW FORMAT SERDE\n",
+    "  'org.openx.data.jsonserde.JsonSerDe'\n",
+    "WITH SERDEPROPERTIES (\n",
+    "  'ignore.malformed.json'='true')\n",
+    "STORED AS INPUTFORMAT\n",
+    "  'org.apache.hadoop.mapred.TextInputFormat'\n",
+    "OUTPUTFORMAT\n",
+    "  'org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat'\n",
+    "LOCATION\n",
+    "  '{QUILT_URL}/.quilt/packages'\n",
+    "TBLPROPERTIES (\n",
+    "  'has_encrypted_data'='false',\n",
+    "  'transient_lastDdlTime'='1605312102')\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73215c0a-50a5-4ca4-b618-e88a995a35fe",
+   "metadata": {},
+   "source": [
+    "### B. Package metadata table\n",
+    "Package names and top hashes are not stored in the manifests. Rather they are stored in pointer files in the `.quilt/named_packages` folder.\n",
+    "The following DDL creates a table from these pointer files to make package\n",
+    "top hashes available in Athena.\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "1ae3532c-a91e-4168-824a-3a73a941f9c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DDL[PACKAGES_TABLE] = f\"\"\"\n",
+    "CREATE EXTERNAL TABLE IF NOT EXISTS `{ATHENA_DB}.{PACKAGES_TABLE}`(\n",
+    "  `hash` string)\n",
+    "ROW FORMAT DELIMITED\n",
+    "  FIELDS TERMINATED BY ',' STORED AS INPUTFORMAT\n",
+    "  'org.apache.hadoop.mapred.TextInputFormat'\n",
+    "OUTPUTFORMAT\n",
+    "  'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'\n",
+    "LOCATION\n",
+    "  '{QUILT_URL}/.quilt/named_packages'\n",
+    "TBLPROPERTIES (\n",
+    "  'has_encrypted_data'='false',\n",
+    "  'transient_lastDdlTime'='1557626200')\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa42355a-6791-4a6c-902e-29b64943abef",
+   "metadata": {},
+   "source": [
+    "### C. View of package-level metadata\n",
+    "The DDL below creates a view that contains package-level information including: \n",
+    "* User\n",
+    "* Package name\n",
+    "* Tophash\n",
```

Comment:
```suggestion
    "* Top hash\n",
```

Reply:
I don't particularly care, but that is how it is referenced elsewhere:

>             raise ValueError(f"Tophash {top_hash} was not found in registry {self.base}")

Reply:
Eh, never mind, we use "top hash" elsewhere.